source/_integrations/ondilo_ico.markdown

```diff
@@ -28,6 +28,20 @@ Sensors will be created for all those data.
 
 {% include integrations/config_flow.md %}
 
+## Polling
+
+The default interval to update all data is 1 hour.
+
+If you want to update in shorter intervals, because an Ondilo ICO measures only once an hour, which may result in a difference of the time a measure has been taken and the time the Home Assistant sensor is updated, you disable automatic polling in the configuration screen of the integration and setup an automation using [homeassistant.update_entity](https://www.home-assistant.io/integrations/homeassistant/#service-homeassistantupdate_entity) service.
```

Comment:
This is not integration specific documentation, and thus should not be documented here.